kytos/core/controller.py

```diff
@@ -251,14 +253,21 @@ def toggle_debug(self, name=None):
     def start(self, restart=False):
         """Create pidfile and call start_controller method."""
         self.enable_logs()
-        if self.options.database:
-            self.db_conn_or_core_shutdown()
-            self.start_auth()
-        if self.options.apm:
-            self.init_apm_or_core_shutdown()
-        if not restart:
-            self.create_pidfile()
-        self.start_controller()
+        # pylint: disable=broad-except
+        try:
+            if self.options.database:
+                self.db_conn_or_core_shutdown()
+                self.start_auth()
+            if self.options.apm:
+                self.init_apm_or_core_shutdown()
+            if not restart:
+                self.create_pidfile()
+            self.start_controller()
+        except Exception as exc:
+            exc_fmt = traceback.format_exc(chain=True)
+            message = f"Kytos couldn't start because of {str(exc)} {exc_fmt}"
+            print(message)
```

Comment:
Just want to double check if this should be `print` instead of `self.log.critical`

Reply:
Hi @italovalcy, yes, initially it was `self.log.exception` (relatively similar to `self.log.critical`), but I had to revert it on https://github.com/kytos-ng/kytos/pull/414/commits/f4432f954b505cadec89658b4785e568b608d239. In summary, at this point here, since we're using `QueueListener` by default and it's threaded, it might end up not ready yet, so even a `self.log.listener.stop()` doesn't guarantee that it'd print out correctly (so it might lose the message before emiting the event or processing it). So, `print` ensure it's deterministic here, similarly to other parts of the code were either the log isn't ready or setup yet https://github.com/kytos-ng/kytos/blob/master/kytos/core/controller.py#L175